src/features/counterfactual/store/undeployedSafeSlice.ts

```diff
@@ -0,0 +1,52 @@
+import { type RootState } from '@/store'
+import { createSelector, createSlice, type PayloadAction } from '@reduxjs/toolkit'
+import type { PredictedSafeProps } from '@safe-global/protocol-kit'
+
+type UndeployedSafeSlice = { [address: string]: PredictedSafeProps }
+
+type UndeployedSafeState = { [chainId: string]: UndeployedSafeSlice }
+
+const initialState: UndeployedSafeState = {}
+
+export const undeployedSafeSlice = createSlice({
+  name: 'undeployedSafe',
```

Comment:
```suggestion
  name: 'undeployedSafes',
```

Our other slices are named in plural

Reply:
done